.chalice/config.json

```diff
@@ -2,11 +2,16 @@
   "version": "2.0",
   "app_name": "zap",
   "stages": {
-    "dev": {
+    "local": {
+      "environment_variables": {
+        "ENV": "local"
+      }
+    },
+    "staging": {
       "autogen_policy": false,
       "iam_policy_file": "policy-dev.json",
       "environment_variables": {
-        "ENV": "dev"
+        "ENV": "staging"
```

Comment:
very risky change imo - can we leave as it is, create a new environment instead and we can have three lambda instances instead, each with it's own policy?